BlueShift-iOS-SDK/InApps/BlueShiftNotificationModalViewController.m

```diff
@@ -113,7 +113,6 @@ - (void)hideFromWindow:(BOOL)animated {
             NSString *fileName = [BlueShiftInAppNotificationHelper createFileNameFromURL: self.notification.notificationContent.banner];
             if (fileName && [BlueShiftInAppNotificationHelper hasFileExist: fileName]) {
                 [BlueShiftInAppNotificationHelper deleteFileFromLocal: fileName];
-                NSLog(@"Image file deleted");
```

Comment:
should we remove existing logs?

Reply:
It looked like this log was just for testing purposes so thought of removing it.